app/templates/app/languages.html

```diff
@@ -0,0 +1,54 @@
+{% extends "base.html" %}
+{% load static %}
+{% load i18n %}
+
+{% block content %}
+<br/>
+<div class="detail">
+  <h2>Languages</h2>
+  <br>
+    <div class="detail-row row">
+        {% for item in language_data %}
+          <h5>{{ item.language.name }}</h5>
+          <div class="col-md-6 col-12 detail-col">
+            <p><strong>Documents</strong></p>
```

Comment:
This is a sub-heading, so please consider using an HTML heading, then no specific styling (like `<strong>`) is needed here. Above it jumped from h2 to h5. Maybe use them in order h1-h2-h3, unless that messes with your design.

Same thing below with Projects.

Please review all these headings to mark for translation as needed.